torch/_custom_op/autograd.py

```diff
@@ -102,7 +103,7 @@ def forward(ctx, *flat_args):
             ctx.set_materialize_grads(True)
             args = pytree.tree_unflatten(list(flat_args), spec)
             with torch._C._AutoDispatchBelowAutograd():
-                output = forward_op(*args)
+                output = op_overload(*args)
```

Comment:
This makes it so that instead of calling the special logic in` __call__` of custom_op, we now call into `torch.ops*` directly. How much should we worry about what the comment says in `__call__` about directly calling into `torch.ops*` doing bad things, or are we saying with the new updated API we don't really care about users calling into the op through`torch.ops.*` anymore?

Reply:
> are we saying with the new updated API we don't really care about users calling into the op throughtorch.ops.* anymore?

I'm going to delete `CustomOp.__call__` at some point. We're telling people to use the torch.ops.blah now. There are some pitfalls with it IF you deregister the operator, but no one actually does that in practice.

Reply:
I see, thanks!